Assets/Scripts/Character/BirdController.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BirdController : MonoBehaviour
+{
+    Rigidbody2D rigidbody2d;
+    public int maxHealth = 10;
+    public int maxHunger = 100;
+    public float forwardSpeed = 4.0f;
+    public float slowSpeed = 1.0f;
+    public float upwardSpeed = 3.0f;
+    public float diveSpeed = 6.0f;
+    public float hungerFrequency = 1.0f;
+    public int health { get { return currentHealth; } }
+    public int hunger { get { return currentHunger; } }
+    private int currentHealth;
+    private int currentHunger = 0;
+    private float horizontal;
+    private float vertical;
+    private bool movementEnabled = true;
+    private float upwardTimer;
+
+    private Animator animator;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        rigidbody2d = GetComponent<Rigidbody2D>();
+        currentHealth = maxHealth;
+        animator = GetComponent<Animator>();
+        upwardTimer = hungerFrequency;
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        horizontal = Input.GetAxis("Horizontal");
+        vertical = Input.GetAxis("Vertical");
+    }
+
+    void FixedUpdate()
+    {
+        if (this.movementEnabled)
```

Comment:
all this should be its own function that takes horizontal and vertical as arugments